.github/workflows/mise.yml

```
@@ -0,0 +1,48 @@
+      postgres:
+        image: postgres:16-alpine
+        ports:
+          - "5432:5432"
+        env:
+          POSTGRES_DB: rails_test
+          POSTGRES_USER: rails
+          POSTGRES_PASSWORD: password
+      redis:
+        image: redis:alpine
+        ports: ["6379:6379"]
+        options: --entrypoint redis-server  
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)